src/js/components/Select/SelectContainer.js

```diff
@@ -390,6 +427,7 @@ class SelectContainer extends Component {
               />
             </Box>
           )}
+          {value && clearValuePosition === 'top' ? this.renderClear() : null}
```

Comment:
would it be better to use undefined instead of null?

Reply:
i think you can't return undefined for a jsx. i ll need to test this

Reply:
i always return null, but undefined would have the same effect https://stackoverflow.com/a/42083262/711672